R/misc_yhat.R

```diff
@@ -143,6 +156,7 @@ yhat.ranger <- function(X.model, newdata, ...) {
 yhat.model_fit <- function(X.model, newdata, ...) {
   if (X.model$spec$mode == "classification") {
     response <- as.matrix(predict(X.model, newdata, type = "prob"))
+    colnames(response) <- X.model$lvl
```

Comment:
would be safer to use `make.names()` function to avoid potential problem with spaces

Reply:
Are we sure about that? Purpose of that line is to make sure that columns have exactly the same names as target variable levels. Otherwise we wouldn't be able to pull out proper column.